docs/kpod-logs.1.md

```diff
@@ -0,0 +1,61 @@
+% kpod(1) kpod-logs - Fetch the logs of a container
+% Ryan Cole
+# kpod-logs "1" "March 2017" "kpod"
+
+## NAME
+kpod logs - Fetch the logs of a container
+
+## SYNOPSIS
+**kpod** **logs** [*options* [...]] container
+
+## DESCRIPTION
+The kpod logs command batch-retrieves whatever logs are present for a container at the time of execution.  This does not guarantee execution order when combined with kpod run (i.e. your run may not have generated any logs at the time you execute kpod logs
+
+## OPTIONS
+
+**--follow, -f**
+
+Follow log output.  Default is false
+
+**--since=TIMESTAMP**
+
+Show logs since TIMESTAMP
+
+**--tail=LINES**
+
+Ouput the specified number of LINES at the end of the logs.  LINES must be a positive integer or 'all'.  Defaults to 'all'
+
+## EXAMPLE
+
+kpod logs b3f2436bdb978c1d33b1387afb5d7ba7e3243ed2ce908db431ac0069da86cb45
+
+2017/08/07 10:16:21 Seeked /var/log/crio/pods/eb296bd56fab164d4d3cc46e5776b54414af3bf543d138746b25832c816b933b/c49f49788da14f776b7aa93fb97a2a71f9912f4e5a3e30397fca7dfe0ee0367b.log - &{Offset:0 Whence:0}
+2017-08-07T10:10:09.055123874-04:00 stdout 1:C 07 Aug 14:10:09.055 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
```

Comment:
We should strip out `2017-08-07T10:10:09.055123874-04:00 stdout`. This is the CRI format requirement. Typically applications will add their own time stamp which should be visible.